BrainPortal/app/helpers/select_box_helper.rb

```diff
@@ -592,5 +592,24 @@ def regroup_users_by_lock_status(users) #:nodoc:
     ordered_by_lock_status
   end
 
+  def writable_data_providers(user=current_user) #:nodoc:
```

Comment:
If this is a helper method then it shouldn’t be :nodoc: ; please provide a few short lines of documentation about what the method returns. Also for the other method below